Fix Logging in ManageOperators Class

mcdonnnj opened the pull request:
# <!-- Use the title to describe PR changes in the imperative mood --> #

## 🗣 Description ##

This PR fixes an issue with a logging message and makes some small edits to improve readability and consistency.
<!-- Describe the "what" of your changes in detail. -->
<!-- To avoid scope creep, limit changes to a single goal. -->

## 💭 Motivation and context ##

I was testing a bash script that uses this today and when using the `add` command twice ran into an error with logging output:

```log
--- Logging error ---
Traceback (most recent call last):
  File "/Users/mcdonnnj/.pyenv/versions/3.9.1/lib/python3.9/logging/__init__.py", line 1079, in emit
    msg = self.format(record)
  File "/Users/mcdonnnj/.pyenv/versions/3.9.1/lib/python3.9/logging/__init__.py", line 923, in format
    return fmt.format(record)
  File "/Users/mcdonnnj/.pyenv/versions/3.9.1/lib/python3.9/logging/__init__.py", line 659, in format
    record.message = record.getMessage()
  File "/Users/mcdonnnj/.pyenv/versions/3.9.1/lib/python3.9/logging/__init__.py", line 363, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/Users/mcdonnnj/.pyenv/versions/manage-cyhy-ops/bin/manage-cyhy-ops", line 33, in <module>
    sys.exit(load_entry_point('manage-cyhy-ops', 'console_scripts', 'manage-cyhy-ops')())
  File "/Volumes/workspace/my-repos/manage-cyhy-ops/src/manage_cyhy_ops/cli.py", line 151, in main
    manager.add_user(username, ssh_key, overwrite=overwrite_ssh_key)
  File "/Volumes/workspace/my-repos/manage-cyhy-ops/src/manage_cyhy_ops/manageoperators.py", line 140, in add_user
    return self._update_cyhy_ops_users(username)
  File "/Volumes/workspace/my-repos/manage-cyhy-ops/src/manage_cyhy_ops/manageoperators.py", line 91, in _update_cyhy_ops_users
    logging.info(log_msg, username, self.region)
Message: 'Successfully performed no operations on CyHy Operators in region "%s"'
Arguments: ('first.last', 'us-east-1')
```

While fixing that issue I improved the output phrasing and made changes to keep all the logging consistent as a result.
<!-- Why is this change required? -->
<!-- What problem does this change solve? How did you solve it? -->
<!-- Mention any related issue(s) here using appropriate keywords such
<!-- as "closes" or "resolves" to auto-close them on merge. -->

## 🧪 Testing ##

Automated testing passes. I verified that the modified version does not exhibit the error mentioned above.

```log
WARNING SSH key for "first.last" already exists in the Parameter Store for region "us-east-1".
WARNING If you need to overwrite this value, please use the "--overwrite" switch.
WARNING User "first.last" is already in the list of active CyHy Operators in region "us-east-1".
INFO Performed no operations for "first.last" in region "us-east-1"
```
<!-- How did you test your changes? How could someone else test this PR? -->

<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->

## ✅ Checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- Draft PRs should have one or more unchecked boxes. -->
<!-- If you're unsure about any of these, don't hesitate to ask. -->
<!-- We're here to help! -->

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

mcdonnnj committed:
Fix logging message in ManageOperators._update_cyhy_ops_users()

The default value does not include an operator for the user value we pass in
the logging output call. This results in a TypeError exception because there
are more arguments than the format string has operators.

mcdonnnj committed:
Update logging outputs for consistency

Since "Successfully" was removed to improve readability for the output in
ManageOperators._update_cyhy_ops_users(), I have mirrored that change in the
rest of the class.

dav3r commented on src/manage_cyhy_ops/manageoperators.py:
```diff
@@ -87,7 +87,7 @@ def _update_cyhy_ops_users(self, username: str, remove: bool = False) -> int:
                 Overwrite=True,
             )
-            log_msg = f'Successfully {update_msg} CyHy Operators in region "%s"'
+            log_msg = f'{update_msg} in region "%s"'
```
Consistently end all log messages (at least from what I see in this PR) with a period.  I leave it as an exercise for the reader to check the rest of the file. 😉 
```suggestion
            log_msg = f'{update_msg} in region "%s".'
```

dav3r reviewed (approved):
LGTM! ⛔ 🐞 
Please note my single-character suggestion.

mcdonnnj committed:
Fix missing period from log message

Consistency is key.

Co-authored-by: dav3r <david.redmin@trio.dhs.gov>

mcdonnnj committed:
Update logging output in ManageOperators class

Further tweak the logging messages to ensure they are consistent in structure
and language.

jsf9k reviewed (approved):
LGTM!

dav3r reviewed (approved):
Solid! 🎸
